default.json

```diff
@@ -22,6 +22,7 @@
       "matchManagers": ["pre-commit"]
     }
   ],
+  "pinDigests": false,
```

Comment:
I think this disables pinning digests in all places, not just Docker.

We pin digests in github actions - are we intentionally turning this off as well here? I think Renovate would still suggest pinning digests for those as part of their best practice.

Reply:
Ah fair, I figured it was only doing Dockerfiles as per the below screenshot:

<img width="653" alt="image" src="https://github.com/user-attachments/assets/95cf7f83-9a1a-4e16-906b-42d2d99f4d1b" />

I'll try find a Dockerfile-scoped alternative

Reply:
👀  i could definitely be confidently wrong...

Reply:
We will never know now 😄